src/Zenova/Helper.cpp

```diff
@@ -26,7 +26,7 @@ namespace Zenova {
 
 		bool run = (PlatformImpl::Init(platformArgs) && !manager.dataFolder.empty());
 		if (run) {
-			logger.info("Zenova Started");
+			logger.info("Zenova Started (Dev)");
```

Comment:
I feel that this is unneeded for this PR

Reply:
yeah, i forgot to remove, it was from my repo xD